alts/src/main/java/io/grpc/alts/internal/ProtectedPromise.java

```diff
@@ -18,6 +18,7 @@
 
 import static com.google.common.base.Preconditions.checkState;
 
+import io.grpc.Internal;
```

Comment:
This is not sufficient.  You need to expose this via a class named Internal...

Reply:
done. PTAL